src/astro/sql/table.py

```diff
@@ -37,7 +37,7 @@ def __str__(self):
 
 
 class TempTable(Table):
-    def __init__(self, conn_id, database, warehouse=""):
+    def __init__(self, conn_id=None, database=None, warehouse=""):
```

Comment:
Which is the use-case to have `conn_id=None`?

Reply:
@tatiana the case where conn_id is none is the case where you inherit the conn_id from the previous task. We might want to decide at runtime what conn_id to use.